packages/sanity/package.json

```diff
@@ -148,6 +148,7 @@
     "@dnd-kit/modifiers": "^6.0.1",
     "@dnd-kit/sortable": "^7.0.2",
     "@dnd-kit/utilities": "^3.2.2",
+    "@isaacs/ttlcache": "^1.4.1",
```

Comment:
I was about to comment how I had some concerns about it being a pretty stale (last release updated was 2 years ago) but then I saw the creator is also the creator of npm, I think it should be fine and that the likelihood of us running into issues is low :) 

(also double checked the license, ISC should be fine since from my understanding it's MIT adjacent)